.travis.yml

```diff
@@ -129,7 +130,12 @@ script:
     - git clone https://github.com/mendersoftware/integration.git
 
     # Run dependant microservices and the tests service itself
-    - TESTS_DIR=$PWD/tests ./tests/run-acceptance $PWD/integration ./tests/docker-compose.yml
+    - TESTS_DIR=$PWD/tests ./tests/run-test-environment "acceptance" $PWD/integration ./tests/docker-compose.yml
+
+    # Run dependant microservices and the tests service itself
+    - NO_CLEANUP=true TESTS_DIR=$PWD/tests ./tests/run-test-environment "integration" $PWD/integration ./tests/docker-compose-integration.yml &
```

Comment:
don't we want compose up to complete before proceeding to run integration tests?